setup/src/Magento/Setup/Test/Unit/Fixtures/CartPriceRulesFixtureTest.php

```diff
@@ -222,7 +222,7 @@ public function testGenerateAdvancedCondition($ruleId, $categoriesArray, $ruleCo
                 'type' => Address::class,
                 'attribute' => 'region',
                 'operator' => '==',
-                'value' => $regions[($ruleId / 4) % 50],
+                'value' => $regions[intdiv($ruleId, 4) % 50],
```

Comment:
Could you explain why it's needed?

Reply:
@ihor-sviziev Thanks for your review. Please check this code [example](https://3v4l.org/Z3Y7P).